config/dune

```diff
@@ -38,4 +38,4 @@
    plugin_list
   ; used for the prefix
   (env_var PWD))
- (action (chdir %{project_root} (run %{project_root}/tools/configure/configure.exe -no-ask))))
+ (action (chdir %{project_root} (run %{project_root}/tools/configure/configure.exe -quiet -prefix ../install/default))))
```

Comment:
This seems wrong?

Should it be `%{workspace_root}/_build/install/default` ?

Reply:
IDK, this is what it was doing before, cf https://github.com/coq/coq/blob/5ce4a881c2306f41367fa952ef96199e95919cde/tools/configure/configure.ml#L285

Reply:
This was working fine because we were instructing the call to configure from the workspace root ? No idea, or likely we were passing `-prefix` by all users due to the above being buggy.

I'm not sure, but certainly assuming the existence of `../install` in rules is a bug, as we fixed recently in other PRs.

Reply:
changed